### ISSUE TITLE
Studio AsyncApi generator calling localhost:3001

### ISSUE THREAD

**Issue by gamingcode775** [asyncapi/studio#1149]
### Describe the bug
Hi team, I tried to use your online studio to generate MD file from your example with version 3.0.0, but it is calling localhost3001 instead of your server.

### To Reproduce
1. Open https://studio.asyncapi.com/
2. Generate md document

### Sample document
If applicable, add an AsyncAPI document illustrating the problem.

### Screenshots
<img width="1112" alt="Screenshot 2024-10-12 at 10 03 33" src="https://github.com/user-attachments/assets/efd51a97-e6ab-4024-afb4-d7aa5c3dfe1e">

**Comment by abhavgoel** [comment]
#1104 - the url was fixed in this, but the generator still fails for spec version 3.0.0

**Comment by Shurtu-gal** [comment]
> [#1104](https://github.com/asyncapi/studio/issues/1104) - the url was fixed in this, but the generator still fails for spec version 3.0.0

Yes a new server-api is being deployed as I write this. Let's hope it gets resolved.
For generator issue: this is the tracking one -- https://github.com/asyncapi/studio/issues/1192
